### ISSUE TITLE
Mark fields as read only when using dump_only

### ISSUE THREAD

**Issue by gpip** [fuhrysteve/marshmallow-jsonschema#27]
marshmallow allows marking fields with dump_only / load_only, it would be good to know fields that have been marked as such. Do you think this would require adding another field to JSONSchema, e.g. read_only / write_only?

**Comment by fuhrysteve** [comment]
I added support for `dump_only` to be `readonly` in the latest master - at a glance I'm not seeing anything in the json schema spec for something resembling write only..

**Comment by gpip** [comment]
I don't think that is part of the json schema. marshmallow uses dump_only to mean read only, and load_only for write only, that's what I was referring to.

**Comment by fuhrysteve** [comment]
Verified against 0.15.0: `fields.X(dump_only=True)` emits `readOnly: True`. Landed in 0.12.0 via #147. Closing.
